vuework04/src/router/index.js

```diff
@@ -18,9 +19,10 @@ const routes = [
     component: Login,
   },
   {
-    path: '/usersView',
-    name: 'UsersView',
-    component: UsersView,
+    path: '/dashboard',
+    name: 'Dashboard',
+    component: Dashboard,
+    meta: {requiresAuth: true},
```

Comment:
`{}`の両内側にはスペースをあけましょう。
linterは色々と設定がいじれますので、調べてみると良いかと思います。
```suggestion
    meta: { requiresAuth: true },
```